promql/engine.go

```diff
@@ -1604,7 +1604,7 @@ func (ev *evaluator) eval(ctx context.Context, expr parser.Expr) (parser.Value,
 		if e.Op == parser.COUNT_VALUES {
 			valueLabel := param.(*parser.StringLiteral)
 			if !model.LabelName(valueLabel.Val).IsValid() {
-				ev.errorf("invalid label name %q", valueLabel)
+				ev.errorf("invalid label name %q", valueLabel.Val)
```

Comment:
Since the String method of valueLabel does the quoting already (maybe weirdly, but this is the reason why we get the double quoting), let's still use it and just not do the extra quoting:

```suggestion
				ev.errorf("invalid label name %s", valueLabel)
```